addons/mail/wizard/mail_compose_message.py

```diff
@@ -530,7 +530,7 @@ def _process_state(self, mail_values_dict):
                 mail_values['state'] = 'cancel'
                 mail_values['failure_type'] = 'mail_email_invalid'
             elif done_emails is not None and not mailing_document_based:
-                done_emails.append(mail_to)
+                done_emails.append((record_id, mail_to))
```

Comment:
this method is adding tuple inside `done_emails`  that contains a list of string (email) => so this change should breaks something (the mass mailing done list) in the case of mass mailing at least

so either we should not do this, or we should adapt the `_get_seen_list` that is called from `_get_done_emails`